spec.md

```diff
@@ -870,8 +870,10 @@ Service Instance to other plans. By modifying parameters, users can change
 configuration options that are specific to a service or plan.
 
 To enable support for the update of the plan, a Service Broker MUST declare
-support per service by including `plan_updateable: true` in its [catalog
-endpoint](#catalog-management).
+support per service by including `"plan_updateable": true` in its [catalog
+endpoint](#catalog-management). If `"plan_updateable": false` is declared,
+Platforms SHOULD NOT make an update request to the Service Broker for any
+Service Instance using the given plan.
```

Comment:
Should the absence of this field mean "false" as well?  Seems like it.

Reply:
Oh, just noticed you did that for bindable using slightly different wording. Any reason we can't do the same here?

Reply:
SHOULD NOT or MUST NOT?

I can't think a reason why we should give a Platform a choice about this. Although, Broker should we prepared to deal with rogue Platforms.

Reply:
The main reason I used SHOULD not MUST was to not introduce a breaking change. Will fix the wording!

Reply:
Do you think the wording today allows for it? At best it seems like its ambiguous and this would be more like a clarifying change since IMO this PR doesn't really change the semantics of the spec - just makes it clear that "false" == absence == don't do it.

Reply:
You don't wanna make it a MUST NOT, eh? :-)

Reply:
I'll very happily make this a MUST NOT if everyone is on board!

Reply:
Done :)